programs/bpf/c/src/deprecated_loader/deprecated_loader.c

```diff
@@ -2,8 +2,9 @@
  * @brief Example C-based BPF program that prints out the parameters
  * passed to it
  */
-#include <solana_sdk.h>
-#include <deserialize_deprecated.h>
+#include <sol_types.h>
+#include <sol_log.h>
+#include <sol_deserialize_deprecated.h>
```

Comment:
totally just a style thing, but did you consider this?
```suggestion
#include <sol/types.h>
#include <sol/log.h>
#include <sol/deserialize_deprecated.h>
```

Reply:
heh, I thought you were in the anti-directory camp

Reply:
I'm in the anti-shift key camp!  So / and - always wins over _ on my keyboard.

Reply:
heh, yeah, old ways.  how about `sol-types.h`?

Reply:
kebab case for C headers isn't really a norm though I think, so snake feels better in this case sadly.  That's why I didn't suggest `sol-deserialize-deprecated.h`.  I can take or leave the change of `sol_` to `sol/`, totally your call